.github/ISSUE_TEMPLATE/issue-report.yaml

```
@@ -0,0 +1,42 @@
+name: "Issue report"
+description: Report documentation issues such as inaccuracies, broken links, typos, or missing information.
+title: "[Issue]: "
+labels: ["Issue"]
```

@elastic/doc-leads a `triage` label? A label that allows us to track how many requests we get from links in the docs?

Something like this?

```suggestion
labels: ["triage", "source:web"]
```

---

Per convo with @shainaraskas, let's add `source:web` via a query param from the docs. This will ensure that issues originating from this repo don't have the label

---

```suggestion
labels: ["triage"]
```

---

Done!